CHANGES.rst

```diff
@@ -47,6 +47,8 @@ New Features
 
   - File name could be passed as ``Path`` object. [#4606]
 
+  - New function ``convenience.table_to_hdu``.
```

Comment:
Could you add the PR number to this entry in the changelog entry? (it will make it easier when we get to the release stage)